manifests/charts/istio-control/istio-discovery/files/injection-template.yaml

```
@@ -46,7 +46,7 @@ metadata:
     {{- end }}
 {{- if .Values.istio_cni.enabled }}
     {{- if not .Values.istio_cni.chained }}
```

You made this suggestion on my PR, do you want to include this here as well?
```suggestion
    {{- if not .Values.istio_cni.chained and eq .Values.cni.provider "multus" }}
```

---

No